math/binomialcoefficient_test.go

```diff
@@ -0,0 +1,39 @@
+// binomialcoefficient_test.go
+// description: Returns C(n, k)
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see binomialcoefficient.go
+
+package math_test
+
+import (
+	"testing"
+
+	algmath "github.com/TheAlgorithms/Go/math"
+)
+
+func TestC(t *testing.T) {
+	var tests = []struct {
+		n        int
+		k        int
+		expected int
+	}{
+		{5, 2, 10},
+		{7, 4, 35},
+		{0, 0, 1},
+		{-1, 1, -1},
+		{1, -1, -1},
+		{-1, -1, -1},
+	}
+	for _, test := range tests {
+		result, error := algmath.C(test.n, test.k)
+		t.Log(test.n, " ", result, " ", error)
+		if result != test.expected {
+			t.Errorf("Wrong result! Expected:%v, Returned:%v, Error:%v", test.expected, result, error.Error())
+		}
+	}
+}
+func BenchmarkBinomialCoefficient(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		algmath.C(65536, 65536)
```

Comment:
This comes up because you have two return values and one of which is an `error` type. You can just write it like this

```suggestion
		_, _ = algmath.C(65536, 65536)
```
which would make it clear that they are just ignored and not forgotten to the linter.

Reply:
Fixed [here](https://github.com/TheAlgorithms/Go/pull/539/commits/9d063f38d84e2228b8a476ec3ec0a537e35e01b5)